themes-default/slim/static/js/templates/config-textbox-number.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <div id="config-textbox-number-content">
+        <div class="form-group">
+            <div class="row">
+                <label :for="name" class="col-sm-2 control-label">
+                    <span>{{ label }}</span>
+                </label>
+                <div class="col-sm-10 content">
+                    <input type="number" :min="min" :step="step"  :id="id" :name="id" :value="localValue" :class="inputClass"/>
```

Comment:
Checkout v-bind for this as it'd allow you to replace `:min="min" :step="step"  :id="id"` with `v-bind='{ min, id, step }`.